nextflow_schema.json

```
@@ -109,6 +109,10 @@
                     "description": "Naming of sequencing files",
                     "help_text": "Indicates the naming of sequencing files (default: `\"/*_R{1,2}_001.fastq.gz\"`).\n\nPlease note:\n\n1. The prepended slash (`/`) is required\n2. The star (`*`) is the required wildcard for sample names\n3. The curly brackets (`{}`) enclose the orientation for paired end reads, seperated by a comma (`,`).\n4. The pattern must be enclosed in quotes\n\nFor example for one sample (name: `1`) with forward (file: `1_a.fastq.gz`) and reverse (file: `1_b.fastq.gz`) reads in folder `data`:\n\n```bash\n--input \"data\" --extension \"/*_{a,b}.fastq.gz\"\n```"
                 },
+                "addsh": {
+                    "type": "boolean",
+                    "description": "If ASVs should be assigned to SHs. Only relevant for ITS data."
```

Just to add here the `species hypothesis` info, not only the abbreviation SH.
```suggestion
                    "description": "If ASVs should be assigned to species hypotheses (SHs). Only relevant for ITS data."
```

---

I'd say "UNITE species hypotheses (SHs)"

---

Sounds good! Also, googling that words lead to a explanatory hit, thats desirable I think ;)

---

You're both right, thanks for pointing that out!